gno.land/pkg/keyscli/run.go

```diff
@@ -1,28 +1,29 @@
-package client
+package keyscli
 
 import (
 	"context"
 	"flag"
 	"fmt"
-	"io/ioutil"
+	ioutil "io"
```

Comment:
```suggestion
	"io"
```

Reply:
I didn't use io because it will make me rename 'io' argument from commands.IO

Reply:
do rename that one ;)

Reply:
done !